app/javascript/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { Provider } from "react-redux";
+import Greeting from "./Greeting";
+import configureStore from "../configureStore";
+const store = configureStore();
+
+class App extends React.Component {
+  render() {
+    return (
+      <Provider store={store}>
+        <BrowserRouter>
+          <Routes>
+            <Route path="/" element={<Greeting />} />
+            {/* <Route path="/greeting" element={<Greeting greeting={"greeting Friend "} />} /> */}
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.